### ISSUE TITLE
New translation: {Indonesian}

### ISSUE THREAD

**Issue by maengkom** [squidfunk/mkdocs-material#901]
## Instructions

1. Check, if your language is already available: [here](http://bit.ly/2DCzaL0)
2. If it isn't, please translate the labels on the right:

``` jinja
{% macro t(key) %}{{ {
  "language": "id",
  "clipboard.copy": "Salin ke memori",
  "clipboard.copied": "Tersalin ke memori",
  "edit.link.title": "Ubah halaman ini",
  "footer.previous": "Sebelumnya",
  "footer.next": "Selanjutnya",
  "meta.comments": "Komentar",
  "meta.source": "Sumber",
  "search.language": "id",
  "search.placeholder": "Cari",
  "search.result.placeholder": "Ketik untuk mulai pencarian",
  "search.result.none": "Tidak ada dokumen yang sesuai",
  "search.result.one": "1 dokumen ditemukan",
  "search.result.other": "# dokumen ditemukan",
  "search.tokenizer": "[\s\-]+",
  "skip.link.title": "Lewati ke isi",
  "source.link.title": "Menuju repositori",
  "toc.title": "Daftar isi"
}[key] }}{% endmacro %}
```

Thanks!

**Comment by squidfunk** [comment]
Awesomeness thanks! I've been traveling Indonesia for the last three weeks, leaving today for my trip back home from Jakarta 😊

**Comment by maengkom** [comment]
@squidfunk wow... I hope you enjoyed it and will be back in the future 👍

**Comment by squidfunk** [comment]
Added as part of 793263f.

**Comment by squidfunk** [comment]
Released as part of 3.0.5
